tokio/CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ Biggest changes are:
 - fs: `File` operations take `&self` (#2930).
 - rt: runtime API, and `#[tokio::main]` macro polish (#2876)
 - rt: `Runtime::enter` uses an RAII guard instead of a closure (#2954).
+- net: the `from_std` function on all sockets no longer sets `O_NONBLOCK` (#2893)
```

Comment:
```suggestion
- net: the `from_std` function on all sockets no longer sets socket into non-blocking mode (#2893)
```

"non-blocking mode" is may easier to understand, given that the tokio, [mio](https://docs.rs/mio/0.7.3/mio/net/struct.TcpListener.html#method.from_std), and [std](https://doc.rust-lang.org/nightly/std/net/struct.TcpStream.html#method.set_nonblocking) docs don't mention `O_NONBLOCK` directly.